XamlControlsGallery/PageHeader.xaml

```diff
@@ -75,6 +75,26 @@
                     IsOpen="False"
                     IsSticky="False"
                     Visibility="Collapsed">
+                    <AppBarButton x:Name="CopyLinkButton" Click="OnCopyLinkButtonClick">
+                        <AppBarButton.Icon>
+                            <FontIcon x:Name="CopyLinkButtonIcon" Margin="0,-2,0,0" Glyph="&#xE71B;" />
```

Comment:
Could this be replaced with a SymbolIcon with Symbol="Copy"?

Reply:
The checkmark is part of MDL2, but not part of the Symbol Icon enumeration, so using FontIcon per [guidance](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.symbol?view=winrt-19041#remarks). Could update to a different set of icons, if you think that would be better.

Reply:
If we want to use the "SymbolIcon" Copy, we can use  glyph `&#xE8C8;` but since copy is available as symbol, we can also use a SymbolIcon here.

Reply:
Are there any technical reasons to prefer SymbolIcon over FontIcon in this case? IMO, the checkmark best conveys the confirmation that the generated link was copied to clipboard, and it aligns with the feedback above (an earlier iteration used the Copy glyph), so if there is not a strong reason for choosing over the other, I think I'd prefer to keep it FontIcon to enable this experience (unless you all disagree, of course, and want to revert back to the original copy UX):

![image](https://user-images.githubusercontent.com/1414423/105905442-4aa70800-5fd7-11eb-9b17-0b460dc10818.png)

![image](https://user-images.githubusercontent.com/1414423/105905463-51ce1600-5fd7-11eb-9299-b5c943909e10.png)

Reply:
Oh, thank you for clarifying that this is the check glyph. My preference for SymbolIcon is from a readability point of view, not a technical one. I agree that checkmark seems appropriate here.

FWIW, it looks like the check glyph is called "Accept" in the SymbolIcon enumeration - maybe that could be used?

Reply:
> it looks like the check glyph is called "Accept" in the SymbolIcon enumeration

Ha - yep! Inconsistent naming across MDL2 and SymbolIcon enum. Happy to make this change.